docs/release-notes/RELEASE-NOTES-0.11.0.md

```diff
@@ -14,6 +14,10 @@ This release introduces backward incompatible changes in the chain format:
 * After upgrading your node, you will not have your previous balance (even if you keep your key pair);
 * Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.10.x".
 
+Configuration related changes for this release (epoch.yaml):
+* The (`http` > `external` > `peer_address`) option has been removed, only the port option (`http` > `external` > `port`) is required if it differs from the internal port.
+* The configuration field (`sync` > `external_port`) has been added for the cases where the publicly accessible port is different from the internal port.
```

Comment:
Distinct section for configuration changes is not in line with current approach - e.g. see https://github.com/aeternity/epoch/blob/214908f39c986d1d70bfbf64a971a435ffa6387c/docs/release-notes/RELEASE-NOTES-0.11.0.md#about-this-release . I suggest rather adding a bullet point above, close to the "Does this" and "Does that" placeholders.